webroot/src/pages/LicensingDetail/LicensingDetail.ts

```diff
@@ -32,14 +32,18 @@ export default class LicensingDetail extends Vue {
     //
     // Computed
     //
-    get licenseStore(): any {
-        return this.$store.state.license;
+    get compact(): string {
+        const defaultCompact = this.$store.state.user.currentCompact;
```

Comment:
How is this different from `defaultCompact` defined in `Home.ts`?

Reply:
I think I was just confused by there being multiple "default" compacts in the code, but it seems they are mostly unrelated. Ignore me!